beta-src/src/webDiplomacyTheme.ts

```diff
@@ -50,10 +50,18 @@ declare module "@mui/material/styles" {
  * define constants that are to be re-used in the theme. use generic names that
  * describe their usage.
  */
+const mainColor = "#000";
+const secondaryColor = "#fff";
 const activeButtonStyle = {
-  backgroundColor: "#fff",
+  backgroundColor: secondaryColor,
   boxShadow: "0 0 2px 2px #000",
-  color: "#000",
+  color: mainColor,
+};
+
+const focusButtonStyle = {
+  backgroundColor: mainColor,
+  boxShadow: "0 0 2px 2px #fff",
+  color: secondaryColor,
```

Comment:
same thing here for box-shadow. leave color off and it will be secondaryColor by default